src/renderer/components/dialogs/MailtoDialog/index.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react'
+import AutoSizer from 'react-virtualized-auto-sizer'
+import { C } from '@deltachat/jsonrpc-client'
+
+import ChatListItem from '../../chat/ChatListItem'
+import { PseudoListItemNoSearchResults } from '../../helpers/PseudoListItem'
+import { ChatListPart, useLogicVirtualChatList } from '../../chat/ChatList'
+import { useChatList } from '../../chat/ChatListHelpers'
+import { useThemeCssVar } from '../../../ThemeManager'
+import { selectedAccountId } from '../../../ScreenController'
+import Dialog, { DialogBody, DialogContent, DialogHeader } from '../../Dialog'
+import useCreateDraftMessage from '../../../hooks/chat/useCreateDraftMesssage'
+import useTranslationFunction from '../../../hooks/useTranslationFunction'
+
+import styles from './styles.module.scss'
+
+import type { DialogProps } from '../../../contexts/DialogContext'
+
+type Props = {
+  messageText: string
+}
+
+export default function MailtoDialog(props: Props & DialogProps) {
+  const { onClose, messageText } = props
+  const listFlags = C.DC_GCL_FOR_FORWARDING | C.DC_GCL_NO_SPECIALS
+
+  const tx = useTranslationFunction()
+  const createDraftMessage = useCreateDraftMessage()
+  const accountId = selectedAccountId()
+  const { chatListIds, queryStr, setQueryStr } = useChatList(listFlags)
+  const { isChatLoaded, loadChats, chatCache } =
+    useLogicVirtualChatList(chatListIds)
+
+  const onChatClick = async (chatId: number) => {
+    createDraftMessage(accountId, chatId, messageText)
+    onClose()
+  }
+
+  const onSearchChange = (e: React.ChangeEvent<HTMLInputElement>) =>
+    setQueryStr(e.target.value)
```

Comment:
What is the reason one is `async` and the other is not?

Reply:
It was there before this MR